### DISCUSSION TITLE
Rework the HP increases from the Start at Second Campaign option

### DISCUSSION THREAD

**Discussion by DrippingYellow** [DoukutsuRandamu/IssueTracker#90] [Suggestions]
I appreciate the effort to make a second-campaign only run via the Gameplay Options viable, but the system you have in place doesn't seem like it completely offsets the health disadvantage. The current system simply doubles the character's maximum health, which higher-HP characters like Curly (9->18 HP) tend to get more out of than low-HP ones like Toroko (3->6 HP).
No matter which character you choose, however, this is still significantly less health compared to a run going through both campaigns arriving at Stage 11. You're still left about 3 hits from death on average, against enemies that are substantially harder to avoid than in the First Campaign. 

Personally, I think the best way to go about this is to just apply a flat max HP increase across all characters, since characters tend to get the same amount of HP from life capsules as each other. An amount somewhere around 20-30 HP is what I feel would be most fair, while still being just a little less compared to actively gathering life upgrades in the First Campaign.

**Comment by TrashboxBobylev** [comment]
Satisfied in 1.44.1 (by making starting HP be equivalent to 75% of life capsules that would have been collected in first campaign).
